BLSQ/iaso
# IA-3206-IA-3207: Add and move projects filter on completeness and org unit type page

Explain what problem this PR is resolving
- Add projects filter on orgUnit type page
- Move projects filter on completeness stats page
Related JIRA tickets : [IA-3206](https://bluesquare.atlassian.net/browse/IA-3206) [IA-3207](https://bluesquare.atlassian.net/browse/IA-3207)

## Self proofreading checklist


## Doc
- Tell us where the doc can be found (docs folder, wiki, in the code...)

## Changes
- Add projects filter on orgUnit type page
- Move projects filter on completeness stats page

## How to test
- Go to org unit type page and check if it match the [mockup](https://whimsical.com/multi-project-web-interfaces-modifications-LRbazNKxybyqpULUQiKcJx)
- Go to completeness stats page and check if it match the [mockup](https://whimsical.com/multi-project-web-interfaces-modifications-LRbazNKxybyqpULUQiKcJx)

## Print screen / video
![Screenshot-from-2024-08-08-17-30-49](https://github.com/user-attachments/assets/00e83c80-9a0f-4cbd-9f84-b21ab69ebdda)
![Screenshot-from-2024-08-08-17-31-02](https://github.com/user-attachments/assets/b9ea8660-e685-4e13-9d1b-0c6b21f0b4d0)


## Notes

Things that the reviewers should know: known bugs that are out of the scope of the PR, other trade-offs that were made.
If the PR depends on a PR in bluesquare-components, or merges into another PR (i.o. main), it should also be mentioned here


[IA-3206]: https://bluesquare.atlassian.net/browse/IA-3206?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[IA-3207]: https://bluesquare.atlassian.net/browse/IA-3207?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
